s3-eventbridge-glue/example-pattern.json

```diff
@@ -0,0 +1,62 @@
+{
+    "title": "Automatic data transformation with S3, Glue and StepFunctions",
+    "description": "This pattern facilitates the transformation of data by using an event-driven architecture to trigger a data transformation job",
+    "language": "Python",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "When you upload a .csv file to the Input S3 bucket, it matches with an Amazon EventBridge rule that triggers a State Machine. The State Machine is composed of two steps:",
+        "1. Starts a AWS Glue Crawler to crawl the input bucket. This allows you to have the raw data (before transformation) inside the Data Catalog",
+        "2. Start a AWS Glue ETL Job, that runs a simple transformation job and drops empty columns in the dataset" ,
+        "The processed/transformed data is then added to the Data Catalog and to an S3 bucket to allow for further processing."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-eventbridge-glue",
+        "templateURL": "serverless-patterns/s3-eventbridge-glue",
+        "templateFile": "s3-eventbridge-gluecrawler/s3-eventbridge-gluecrawler.py"
```

Comment:
- gitHub.template requires the property "projectFolder". 
- Can you rename the folder `s3-eventbridge-gluecrawler` to `s3-eventbridge-crawler` and then update the `templateFile` accordingly that should all the issues.

Reply:
Resolved